vpr/src/place/net_cost_handler.h

```diff
@@ -1,9 +1,12 @@
 #pragma once
+
```

Comment:
Probably should have an overall comment for this file at the top; can move or reference the .cpp file comment.